scipy/special/add_newdocs.py

```diff
@@ -374,15 +374,15 @@ def add_newdoc(name, doc):
 
     Binomial distribution cumulative distribution function.
 
-    Sum of the terms 0 through `k` of the Binomial probability density.
+    Sum of the terms 0 through `floor(k)` of the Binomial probability density.
 
     .. math::
-        \mathrm{bdtr}(k, n, p) = \sum_{j=0}^k {{n}\choose{j}} p^j (1-p)^{n-j}
+        \mathrm{bdtr}(k, n, p) = \sum_{j=0}^floor(k) {{n}\choose{j}} p^j (1-p)^{n-j}
```

Comment:
Use LaTeX here, i.e. `\lfloor` and `\rfloor`.

Reply:
ok.

Reply:
when I add the \lfloor \rfloor that causes the doc build and therefore the entire build to fail, something like this:" 
```
scipy/signal/_spectral.pyx has not changed
scipy/signal/_sosfilt.pyx has not changed
scipy/stats/_stats.pyx has not changed

Error compiling Cython file:
------------------------------------------------------------
...
    "    events).\n"
    "\n"
    "Returns\n"
    "-------\n"
    "p : ndarray\n"
    "    The event probability such that `bdtr(\\lfloor k 
                                                         ^
------------------------------------------------------------

_ufuncs.pyx:3045:58: Unclosed string literal
Traceback (most recent call last):
  File "/Users/rlucas/scipy-dev/scipy/tools/cythonize.py", line 307, in <module>
    main()
  File "/Users/rlucas/scipy-dev/scipy/tools/cythonize.py", line 303, in main
    find_process_files(root_dir)
  File "/Users/rlucas/scipy-dev/scipy/tools/cythonize.py", line 292, in find_process_files
    for result in pool.imap_unordered(lambda args: process(*args), jobs):
  File "/Users/rlucas/anaconda3/envs/scipy-dev/lib/python3.6/multiprocessing/pool.py", line 761, in next
    raise value
  File "/Users/rlucas/anaconda3/envs/scipy-dev/lib/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/Users/rlucas/scipy-dev/scipy/tools/cythonize.py", line 292, in <lambda>
    for result in pool.imap_unordered(lambda args: process(*args), jobs):
  File "/Users/rlucas/scipy-dev/scipy/tools/cythonize.py", line 231, in process
    processor_function(fromfile, tofile, cwd=path)
  File "/Users/rlucas/scipy-dev/scipy/tools/cythonize.py", line 96, in process_pyx
    raise Exception('Cython failed')
Exception: Cython failed
Cythonizing sources
Traceback (most recent call last):
  File "setup.py", line 540, in <module>
    setup_package()
  File "setup.py", line 524, in setup_package
    generate_cython()
  File "setup.py", line 292, in generate_cython
    raise RuntimeError("Running cythonize failed!")
RuntimeError: Running cythonize failed!

Build failed! (0:00:03.020187 elapsed)
(scipy-dev) Lucass-MacBook:scipy rlucas$ 
```
I only have a single `\` in the docstring in ` scipy/special/add_newdocs.py` but for some reason the doc build is inserting a second `\` into the docstring failing the build.

Reply:
Ah, so that's because the `bdtri` docstring isn't a raw string-make the start of the string `r"""`.

Reply:
ah. that fixed it, thanks! I recall reading about the distinction awhile back but I'll need to re-read the distinction here for sphinx.